src/systems/filecoin_nodes/clock/_index.md

```diff
@@ -2,26 +2,29 @@
 title: Clock
 statusIcon: ✅
 ---
+{{<label clock>}}
 
 {{< readfile file="clock_subsystem.id" code="true" lang="go" >}}
 {{< readfile file="clock_subsystem.go" code="true" lang="go" >}}
 
-Filecoin assumes weak clock synchrony amongst participants in the system. That is, the system relies on participants having access to a globally synchronized clock, tolerating bounded delay in honest clock lower than epoch time (more on this in a forthcoming paper).
 
-Filecoin relies on this system clock in order to secure consensus, specifically ensuring that participants are only running leader elections once per epoch and enabling miners to catch such deviations from the protocol. Given a system start and epoch time by the genesis block, the system clock allows miners to associate epoch and wall clock time, thereby enabling them to reason about block validity and give the protocol liveness.
+Filecoin assumes weak clock synchrony amongst participants in the system. That is, the system relies on participants having access to a globally synchronized clock (tolerating some bounded drift).
```

Comment:
why remove the upper bound on tolerated drift?

Reply:
I stated what seemed to be the same information in the parentheses at the end of the new paragraph "(tolerating some bounded drift)".  I may have cut out information that I didn't understand from reading the old version.  If so let me know and I'll put it back

Reply:
Weird... I thought it stated:
> can tolerate drift up to an epoch length
but can't find it

Ignore unless you think useful